BDArmory/Control/MissileFire.cs

```diff
@@ -2084,11 +2083,10 @@ public int[] GetMissilesAway(TargetInfo target)
 
         private void CalculateMissilesAway() //FIXME - add check for identically named vessels
         {
+            if (!guardMode) return;
             missilesAway.Clear();
             // int tempMissilesAway = 0;
             //firedMissiles = 0;
-            if (!guardMode) return;
```

Comment:
This `(!guardMode) return;` check isn't fail-safe. If for some reason a check for `guardMode` where `MissilesAway` gets used is missing somewhere, this would cause issues under manual operation. I guess the best way to do this would be to do `missilesAway.Clear();` in `ToggleGuardMode()`